e2e_test.go

```diff
@@ -14,7 +14,6 @@ import (
 	"time"
 
 	"github.com/filecoin-project/storetheindex/config"
-	qt "github.com/frankban/quicktest"
```

Comment:
There was an error that `go get` was needed to get the package, yet your PR did not modify go.mod so instead of determining why that complaint only occurred on some platforms, I just eliminated the issue.

I have reverted the change that removed quicktest.